js/backup_restore.js

```diff
@@ -566,7 +566,10 @@ function configuration_restore(callback) {
                     mincheck:               1100,
                     maxcheck:               1900,
                     rx_min_usec:            885,
-                    rx_max_usec:            2115
+                    rx_max_usec:            2115,
```

Comment:
I don't have a problem with it being in there, my remark was more that it's probably not worth putting much effort into maintaining this functionality.